### PR TITLE
WD-6720 - update /internet-of-things/smart-displays

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

- Updated page according to [copydoc](https://docs.google.com/document/d/1fFvBl-ZZvKDFV6OMv7O4kudQKtyxfvSU60rfVdKluAQ/)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check whether the page is according to copy doc

## Issue / Card
[WD-6720](https://warthogs.atlassian.net/browse/WD-6720)

Fixes #

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-6720]: https://warthogs.atlassian.net/browse/WD-6720?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/internet-of-things/smart-displays.html [hunk 1]
```diff
@@ -8,26 +8,30 @@
...
-        <p><a href="/core/services" class="p-button--positive">Build digital signage with IoT Professional Services</a></p>
+        <p class="p-heading--4">The easy, secure, and reliable way of building open source digital signage software for your business.</p>
+        <p><a href="/core/services" class="p-button--positive">Contact us to learn more</a></p>
         <p><a href="/engage/Embedding_IoT_GUI_with_Ubuntu_Frame" class="p-link--inverted">Develop GUIs for IoT with our guide&nbsp;&rsaquo;</a></p>
       </div>
     </div>
   </div>
 </section>
 
+<section class="p-strip">
```
Comment (lyubomir-popov) [inline templates/internet-of-things/smart-displays.html:27]:
```suggestion
<section class="p-strip is-shallow">
```

File: templates/internet-of-things/smart-displays.html [hunk 2]
```diff
@@ -8,26 +8,30 @@
...
+  {% include "shared/_18-04-eol-banner.html" %}
+</section>
+
 <section class="p-strip">
```
Comment (lyubomir-popov) [inline templates/internet-of-things/smart-displays.html:31]:
```suggestion
<section class="p-strip is-shallow">
```

**Review by lyubomir-popov — APPROVED** [review]
couple of suggestions, giving +1 so it can be merged quickly after those are accepted.

File: templates/internet-of-things/smart-displays.html [hunk 3]
```diff
@@ -176,17 +177,16 @@ <h3 class="p-heading--4"><a href="https://ubuntu.com/engage/webinarintrotoframe"
...
 </section>
 
 <section class="p-strip is-bordered">
   <div class="row">
     <div class="col-8">
-      <h2>Partners and players</h2>
-      <p>The <a href="https://snapcraft.io/store">Ubuntu Core app store</a> makes it easy for systems integrators and device makers to find solutions, while enabling OEMs to generate revenues from their devices over the entire lifetime of their products.</p>
-      <p ><a class="p-button js-invoke-modal" href="/internet-of-things/contact-us?product=smart-displays" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Digital Signage Page - Partners and players', 'eventLabel' : 'Get in touch', 'eventValue' : undefined });">Get in touch</a></p>
+      <h2>Partners</h2>
+      <p>The Get in touch with one of our partners to help you build your Digital Signage or Kiosk solution using Ubuntu as the foundation of your technology stack.</p>
```
Comment (jw-can) [inline templates/internet-of-things/smart-displays.html:189]:
```suggestion
      <p>Get in touch with one of our partners to help you build your Digital Signage or Kiosk solution using Ubuntu as the foundation of your technology stack.</p>
```

**Comment by juanruitina** [comment]
LGTM!

**Review by jw-can — APPROVED** [review]

**Review by mtruj013 — APPROVED** [review]
QA looks good to me!

**Pull request merged**